### PR TITLE
Alarms logic & Notification system

### PR DESCRIPTION
**Pull request by deRemo**
merge to avoid future conflicts

### CODE REVIEW

File: app/src/main/java/devs/erasmus/epills/controller/AddPillGeneralActivity.java [hunk 1]
```diff
@@ -262,7 +262,7 @@ protected void onPostExecute(String queryResults) {
                 description_text.setText(drugDescription);
 
             } else {
-                nameLayout.setError("INCORRECT SUBSTANCE NAME");
+                name_text.setError("INCORRECT SUBSTANCE NAME OR NO INTERNET CONNECTION");
```
Comment (jcebel) [inline app/src/main/java/devs/erasmus/epills/controller/AddPillGeneralActivity.java]: please use nameLayout not name_text.
This has a better error message

File: app/src/main/java/devs/erasmus/epills/utils/SetAlarmUtil_DEPRECATED.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import devs.erasmus.epills.broadcast_receiver.AlarmBroadcastReceiver;
+
+/**
+ * Created by Remo Andreoli on 15/11/2017.
+ */
+
+public class SetAlarmUtil_DEPRECATED {
```
Comment (jcebel) [inline app/src/main/java/devs/erasmus/epills/utils/SetAlarmUtil_DEPRECATED.java:21]: If it's deprecated - why keep it?

File: app/src/main/res/layout/pillbox_content.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jcebel) [inline app/src/main/res/layout/pillbox_content.xml]: Delete this one. I think pill_box_item substitutes it.

File: app/src/main/res/layout/activity_pill_box.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jcebel) [inline app/src/main/res/layout/activity_pill_box.xml]: Delete this one. I think activity_medicine_box substitutes it

File: app/src/main/java/devs/erasmus/epills/utils/SetAlarmUtil_DEPRECATED.java (hunk 2 above)
Comment (deRemo) [reply app/src/main/java/devs/erasmus/epills/utils/SetAlarmUtil_DEPRECATED.java:21]: i will delete it later, just in case i will re-use it

Comment (deRemo) [reply app/src/main/java/devs/erasmus/epills/utils/SetAlarmUtil_DEPRECATED.java:21]: but now that i think about it the new SetAlarmUtil seems stable, so i will do it now

**Review by jcebel — APPROVED** [review]

**Pull request merged**